### PR TITLE
Updating the banner to match the main page

### PR DESCRIPTION
**Pull request by sfshaza2**
Waiting for the main page banner (and the new page) to go live.

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -5,6 +5,6 @@
     https://github.com/flutter/website/issues/3897
   {% endcomment %}
   <div class="site-banner site-banner--default" role="alert">
-    The next Flutter challenge is here! Join the <a href="https://flutterhack.devpost.com/">Flutter Puzzle Hack</a> for a chance to win prizes.
+    Exciting news for Windows developers&emdash;Flutter is on the way! <a href="flutter.dev/events/flutter-update-windows">Learn more</a>
```
Comment (parlough) [inline src/_includes/banner.html:8]:
```suggestion
    Exciting news for Windows developers&emdash;Flutter is on the way! <a href="https://flutter.dev/events/flutter-update-windows">Learn more</a>
```

**Review by parlough — COMMENTED** [review]
Updating the link fixed the tests and the page seems to be live now, so looks good to me :)

File: src/_includes/banner.html [hunk 2]
```diff
@@ -5,6 +5,6 @@
     https://github.com/flutter/website/issues/3897
   {% endcomment %}
   <div class="site-banner site-banner--default" role="alert">
-    The next Flutter challenge is here! Join the <a href="https://flutterhack.devpost.com/">Flutter Puzzle Hack</a> for a chance to win prizes.
+    Exciting news for Windows developers&emdash;Flutter is on the way! <a href="https://flutter.dev/events/flutter-update-windows">Learn more</a>
```
Comment (kwalrath) [inline src/_includes/banner.html:8]:
```suggestion
    Exciting news for Windows developers—Flutter is on the way! <a href="https://flutter.dev/events/flutter-update-windows">Learn more</a>
```

Comment (kwalrath) [reply src/_includes/banner.html:8]: I finally staged the PR and see this:

image

**Review by kwalrath — CHANGES_REQUESTED** [review]
I think it should be `&mdash;` or `—`, not `&emdash;`.

**Review by kwalrath — APPROVED** [review]
Approved, assuming my change fixes the banner. (Someone please confirm if I don't have time to!)

**Review by domesticmouse — APPROVED** [review]

**Review by domesticmouse — APPROVED** [review]

**Review by khanhnwin — APPROVED** [review]

**Review by timsneath — APPROVED** [review]

**Pull request merged**